index.html

```diff
@@ -15,11 +15,16 @@
 <body>
```

Comment:
- [ ] I kindly suggest that you make use of percentages for your width because it'll prevent the current horizontal scroll because it'll keep a responsive width for all devices.

## FOR EXAMPLE
- Instead of applying 1444px of width to your body selector, you can make use of 100% if the body need 100% of the width on the page.
- Using a width of 1444px would make make your page have an horizontal scroll on devices lower than 1444px

https://www.loom.com/share/a308e6b633684652b15a858c7f8b6ec1

https://www.loom.com/share/707969ee25cc43f18ec74443b5466216